AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -2326,9 +2326,10 @@ lokalavisen.dk,politiken.dk,ekstrabladet.dk#$#body.qc-cmp-ui-showing { overflow:
 ! https://github.com/AdguardTeam/AdguardFilters/issues/16687
 finn.no##.ReactModalPortal
 finn.no#$#body.modal--document-body { overflow: visible!important; }
-! https://github.com/AdguardTeam/AdguardFilters/issues/29104
+! https://github.com/AdguardTeam/AdguardFilters/issues/34920
 !+ PLATFORM(ios)
 nomanssky.com##.cli-show
+theverge.com##.m-privacy-consent
```

Comment:
If it should only apply to iOS, you need to add an additional !+ PLATFORM(ios) above the rule.

Reply:
Yes, it's only for iOS.